src/sound_tool.c

```diff
@@ -0,0 +1,1267 @@
+#include "constants/snd_misc.h"
+#include "constants/snd_system.h"
+#include "constants/snd_def.h"
+#include "sound.h"
+#include "sound_tool.h"
+
+static u8 auto_sample_flag;
+static u32 unk4;
+ 
+typedef struct UnkMicStruct {
+    BOOL mono_flag;
+     u32 unkc;
+} UNK_MICSTRUCT;
+
+static UNK_MICSTRUCT _021D05E8; 
+
+extern MICAutoParam _021D05F8;
+
+#define SWAVE_BUFFER_SIZE 2000
+extern s8 sWaveBuffer[SWAVE_BUFFER_SIZE];
+
+extern u8 _020F5710[8][4];
+
+extern volatile u16 GS_SeqPairs[136][2];
```

Comment:
why volatile? should be const

Reply:
It was labeled volatile because that's the only thing that made it match in CE, but looks like it works proper now in the repo, thanks for the catch!